src/quo/components/tabs/tabs/view.cljs

```diff
@@ -118,87 +119,99 @@
+        set-initial-scroll-poisition      (rn/use-callback (fn []
+                                                             (js/setTimeout
+                                                              (fn []
+                                                                (.scrollToIndex
+                                                                 @flat-list-ref
+                                                                 #js
+                                                                  {:animated false
+                                                                   :index
+                                                                   (utils.collection/first-index
+                                                                    #(= active-tab-id (:id %))
+                                                                    data)}))
+                                                              1)))]
+    (if scrollable?
+      [rn/view {:style {:margin-top (- (dec unread-count-offset))}}
+       [masked-view-wrapper
+        {:fade-end-percentage (get fading :fade-end-percentage) :fade-end? fade-end?}
+        [(if in-scroll-view?
+           gesture/flat-list
+           rn/flat-list)
+         (merge
+          (dissoc props
+           :default-active
+           :fade-end-percentage
+           :fade-end?
+           :on-change
+           :scroll-on-press?
+           :size)
+          {:ref set-flat-list-ref
+           :style style
+           ;; The padding-top workaround is needed because on Android
+           ;; {:overflow :visible} doesn't work on components inheriting
+           ;; from ScrollView (e.g. FlatList). There are open issues, here's just one about this
+           ;; topic: Https://github.com/facebook/react-native/issues/31218
+           :content-container-style
+           (assoc container-style :padding-top (dec unread-count-offset))
+           :horizontal true
+           :scroll-event-throttle 64
+           :shows-horizontal-scroll-indicator false
+           :data data
+           :key-fn (comp str :id)
+           :on-scroll-to-index-failed identity
+           :on-scroll (partial on-scroll-handler
+                               {:fade-end-percentage fade-end-percentage
+                                :fade-end?           fade-end?
+                                :fading              fading
+                                :set-fading          set-fading
+                                :on-scroll           on-scroll})
+           :on-layout set-initial-scroll-poisition
+           :render-fn tab-view
+           :render-data {:active-tab-id       active-tab-id
+                         :set-active-tab-id   set-active-tab-id
+                         :blur?               blur?
+                         :customization-color customization-color
+                         :flat-list-ref       @flat-list-ref
```

Comment:
why would it need to re-render?

Reply:
because when it will be rendered first time @flat-list-ref will be nil

Reply:
hmm yeah, usually `forwardRef` is used for this, but it's kinda unnecessary to pass the ref here anyways. Better just create a `on-press` handler in the parent and pass it down instead of the ref directly.